app/main/index.js

```diff
@@ -56,6 +58,31 @@ const targetURL = function () {
 	return data.domain;
 };
 
+function serverError(targetURL) {
+	if (targetURL.indexOf('localhost:') < 0 && data.domain) {
+		const req = https.request(targetURL + '/static/audio/zulip.ogg', res => {
+			console.log('Server StatusCode:', res.statusCode);
+			console.log('You are connected to:', res.req._headers.host);
+			if (res.statusCode >= 500 && res.statusCode <= 599) {
+				return dialog.showErrorBox('SERVER IS DOWN!', 'We are getting a ' + res.statusCode + ' error status from the server ' + res.req._headers.host + '. Please try again after some time or you may switch server.');
+			}
+		});
+		req.on('error', e => {
+			console.error(e);
+		});
+		req.end();
+	} else if (data.domain) {
+		const req = http.request(targetURL + '/static/audio/zulip.ogg', res => {
```

Comment:
I tried https, It doesn't work for localhost:9991,
so that else if condition is only to avoid error during switching to localhost or loading localhost directly.
I hope I am clear :)